python/dllib/src/bigdl/dllib/nncontext.py

```diff
@@ -488,7 +483,8 @@ def _read_stream(fd, fn):
         # and greater than a single system page.
         buff = fd.read(8192)
         if buff:
-            fn(buff.decode('utf-8'))
+            # fn(buff.decode('utf-8'))
+            fn(buff)
```

Comment:
why change the decode?

Reply:
Any update on the above question?

Reply:
The content is read from log4j file and it's using ascii format not using utf-8 encoding, so don't need decode here?